pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1232,8 +1232,8 @@
       ],
       "BaselineVersion": "4.5.0",
       "InboxOn": {
-        "net45": "4.0.0.0",
         "netcoreapp3.0": "4.0.0.0",
+        "net45": "4.0.0.0",
```

Comment:
>"4.0.0.0" [](start = 17, length = 9)

does the ordering of these items make any difference? just curious to know :-)

Reply:
The DOM will maintain a stable order.  If someone manually changes them then the next person that runs the task will notice the reordering.  It's best to keep them in the order that the DOM uses as it limits noise when folks autogenerate.